content/en/docs/application-store/built-in-apps/postgresql-app.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "PostgreSQL App"
+keywords: 'Kubernetes, KubeSphere, PostgreSQL, app-store'
+description: 'How to use built-in PostgreSQL'
+
+
+weight: 2242
+---
+PostgreSQL is a powerful, open source object-relational database system which is famous for reliability, feature robustness, and performance. This guide will show you one-click deployment for PostgreSQL in Kubenetes .
```

Comment:
Add an external link with the PostgreSQL website here.

Reply:
Got it